tests/test_rules.py

```diff
@@ -317,7 +317,7 @@ def test_from_override_rules_deprecation_using_OverrideRule() -> None:
         "Use 'from_apply_rules' instead."
     )
     with pytest.warns(DeprecationWarning, match=msg):
-        registry = PageObjectRegistry.from_override_rules(rules)
+        registry = RulesRegistry.from_override_rules(rules)
```

Comment:
An option is to change the API to be `RulesRegistry(rules)`, if dict-like API is removed/deprecated. 

I think that's fine to do it later, as a separate step, because we can do it in a backwards-compatible way.

Reply:
Implemented in https://github.com/scrapinghub/web-poet/pull/96/commits/fc8c09139c9f32d4decd42055ee6794ee862a664